YoloDev.Expecto.TestSdk.sln

```diff
@@ -1,14 +1,15 @@
+
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.30114.128
+# Visual Studio 15
+VisualStudioVersion = 15.0.26124.0

```

Comment:
Why?

Reply:
No real reason - part of the revert. I can remove this part of revert.